core/templates/domain/learner_dashboard/learner-dashboard-icons.directive.html

```diff
@@ -43,4 +43,8 @@
     max-width: none;
     white-space: nowrap;
   }
+  .oppia-dropdown-menu {
```

Comment:
This is too generic a name. Make the names specific to the page please, else it might clash with styles from elsewhere.